controllers/categories_controller.js

```diff
@@ -0,0 +1,13 @@
+const normalizeArr = require("../utils/normalize_arr");
+
+function CategoriesController(connection) {
```

Comment:
param `connection` is not used

Reply:
In the future it will be used, when i'll add model

Reply:
remove it for now.